packages/app/components/feed/feed-item-tap-gesture.web.tsx

```diff
@@ -26,38 +27,119 @@ type FeedItemTapGestureProps = {
   children: JSX.Element;
   toggleHeader?: () => void;
   showHeader?: () => void;
+  videoRef?: any;
+  sizeStyle?: {
+    width?: number;
+    height?: number;
+  };
+  mimeType?: string;
 };
-export const FeedItemTapGesture = ({ children }: FeedItemTapGestureProps) => {
+export const FeedItemTapGesture = ({
+  children,
+  videoRef,
+  sizeStyle,
+  mimeType,
+}: FeedItemTapGestureProps) => {
   const { like } = useLike();
 
   const heartAnimation = useSharedValue(0);
+  const playAnimation = useSharedValue(0);
 
   const heartStyle = useAnimatedStyle(() => {
     return {
       opacity: heartAnimation.value,
       transform: [{ scale: heartAnimation.value }],
     };
-  }, [heartAnimation.value]);
+  }, [heartAnimation]);
 
-  const doubleTapHandle = Gesture.Tap()
-    .numberOfTaps(2)
-    .onEnd(() => {
-      heartAnimation.value = withSequence(
+  const playStyle = useAnimatedStyle(() => {
+    return {
+      opacity: playAnimation.value,
+      transform: [{ scale: playAnimation.value }],
+    };
+  }, [playAnimation]);
+
+  const toggleVideoPlayback = useCallback(async () => {
+    const curVideo = videoRef?.current?._nativeRef?.current?._video;
+    if (!curVideo) return;
+
+    if (curVideo.muted) {
+      curVideo.muted = false;
+      return;
+    }
```

Comment:
This could use the mute provider, but Alex want to have the mute icon removed completely. I could still use it though and just output `null` for MuteButton

Reply:
i think mute provider state is also for global muted state and not just icon hide state, right?